src/Services/InvoiceItemService.php

```diff
@@ -99,4 +99,37 @@ public function create($data, $invoiceId, $invoiceItemId = null)
             ['invoiceId' => $invoiceId]
         );
     }
+
+    /**
+     * @param array|JsonSerializable|InvoiceItem $data
+     * @param string $invoiceId
+     * @param string $invoiceItemId
+     *
+     * @throws DataValidationException The input data does not valid
+     *
+     * @return InvoiceItem
+     */
+    public function update($data, $invoiceId, $invoiceItemId)
+    {
+        return $this->client()->put(
+            $data,
+            'invoices/{invoiceId}/items/{invoiceItemId}',
+            ['invoiceId' => $invoiceId, 'invoiceItemId' => $invoiceItemId]
+        );
+    }
+
+    /**
+     * @param array|JsonSerializable|InvoiceItem $data
+     * @param string $invoiceId
+     * @param string $invoiceItemId
+     *
+     * @return InvoiceItem
```

Comment:
We cannot return InvoiceItem as we just deleted it

Reply:
Good catch. Thanks